airflow/kubernetes/pod_launcher.py

```diff
@@ -218,14 +218,16 @@ def _extract_xcom(self, pod: V1Pod):
 
     def _exec_pod_command(self, resp, command):
         if resp.is_open():
-            self.log.info('Running command... %s\n', command)
+            self.log.info('Running command... %s', command)
             resp.write_stdin(command + '\n')
             while resp.is_open():
                 resp.update(timeout=1)
                 if resp.peek_stdout():
-                    return resp.read_stdout()
+                    stdout = resp.read_stdout()
+                    self.log.info("Stdout: %s", stdout.rstrip())
```

Comment:
This is going to "duplicate" the XCom result I think - once from the command output, and again from the log on line 134.

Reply:
I updated this. Here is sample output
<img width="1119" alt="Screenshot 2019-12-09 at 00 06 38" src="https://user-images.githubusercontent.com/12058428/70398100-36d9c480-1a18-11ea-9174-4739aea8ee20.png">